include/pmacc/random/distributions/uniform/Range.hpp

```diff
@@ -41,24 +41,28 @@ namespace uniform
     struct ExcludeZero
     {};
 
-    /**  floating point number in the range (0,1]
+    /**  floating point number in the range [0,1)
      *
      * @tparam T_Type type of the result
      */
     template<typename T_Type>
     struct ExcludeOne
     {
 
-        /** Reduce the random range to 2^24.
+        /** Reduce the random range
+         *
+         * range for `float` is `2^24`
```

Comment:
- remove `` from float or add to double?
- 2^24 is the no. of uniquely different floats? (This is not really the "range".)